deployment/build.gradle

```diff
@@ -61,6 +78,17 @@ buildDeb {
     preUninstall file('src/deb/preUninstall.sh')
 }
 
+docker {
+    name "gregjan/trellis-ext-db:${trellisVersion}"
```

Comment:
can you put `gregjan` into a variable (in the `ext` section above) with a note to change it to `trellisldp` at some point.